Migrations.cs

```diff
@@ -427,5 +427,44 @@ public int UpdateFrom9()
 
             return 10;
         }
+
+        public int UpdateFrom10()
+        {
+            _contentDefinitionManager.AlterPartDefinition("LinkVisualPart", part => part

```

Comment:
Because you're updating an existing part that we know is going to be there, you don't need to specify all the migration again. You only have to define what's being changed, which in this case is the `TextFieldPredefinedListEditorSettings`.

````
_contentDefinitionManager.AlterPartDefinition("LinkVisualPart", part => part
    .WithField("Style", field => field
        .WithSettings(new TextFieldPredefinedListEditorSettings
        {
            Editor = EditorOption.Dropdown,
            Options = new ListValueOption[] {
                new ListValueOption {
                    Name = "Default",
                    Value = string.Empty
                },
                new ListValueOption {
                    Name = "Primary Button",
                    Value = "btn btn--primary"
                },
                new ListValueOption {
                    Name = "Secondary Button",
                    Value = "btn btn--secondary"
                }
            }
        })
    )
);
```